src/cmd/src/bin/greptime.rs

```diff
@@ -62,10 +62,36 @@ enum SubCommand {
 #[global_allocator]
 static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
 
-#[tokio::main]
-async fn main() -> Result<()> {
-    setup_human_panic();
-    start(Command::parse()).await
+fn main() -> Result<()> {
+    // Set the stack size to 8MB for the thread so it wouldn't crash on large stack usage in debug mode
+    // see https://github.com/GreptimeTeam/greptimedb/pull/4317
+    // and https://github.com/rust-lang/rust/issues/34283
+    let builder = std::thread::Builder::new().name("main_larger_stack".to_string());
```

Comment:
Can we use `main` for the thread name?

Reply:
> Can we use `main` for the thread name?

Will cause confusion since the old main function is also named `main` thread